src/Nethermind/Nethermind.History/HistoryPruner.cs

```diff
@@ -29,6 +29,7 @@ public class HistoryPruner : IHistoryPruner
 {
     private const int MaxOptimisticSearchAttempts = 3;
     private const int LockWaitTimeoutMs = 100;
+    private const int SlotsPerEpoch = 32;
```

Comment:
Might make more sense to have it configurable as it may differ across the chains. `IMergeConfig.SlotsPerEpoch` or so. With some mentions what it may affect.